tests/validation/tests/v3_api/test_create_ha.py

```diff
@@ -493,7 +493,17 @@ def create_aks_cluster():
                               'client_id': client_id,
```

Comment:
If we are removing these variables from the terraform files shouldn't these also be removed?

`client_id`
`client_secret`

There are placeholders now that are replaced with the client ID and secret.

Reply:
I think those should be safe to remove, I'll test out locally and check.